PhotoEnhancer/PhotoEnhancer/Filters/Transform/HorizontalTiltParameters.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    class HorizontalTiltParameters: IParameters
+    {
+        public double AngleInDegrees { get; set; }
+
+        public ParameterInfo[] GetDescription()
+        {
+            return new[]
+            {
+                new ParameterInfo() {
+                    Name = "Угол поворота в °",
```

Comment:
Название параметра неудачное. Вводит в заблуждение. Здесь нет поворота.
```suggestion
                    Name = "Угол наклона в °",
```